source/_docs/z-wave.markdown

```diff
@@ -180,6 +180,47 @@ automation:
         scene_id: 11
 ```
 
+Some devices (like the HomeSeer wall switches) allow you to do things like double, and triple click the up and down buttons and fire an event.  This does support the OpenZwave CentralScene handler.  This is an example of double clicking the on/up button:
+
+```yaml
+# Example configuration.yaml automation entry
+automation
+  - alias: 'Dining room dimmer - double tap up'
+    trigger:
+      - event_data:
+          entity_id: light.diningroomcans_level
+          scene_id: 1
+          scene_data: 3
+        event_type: zwave.scene_activated
+        platform: event
+    action:
+        service: homeassistant.turn_on
+        entity_id: group.kitchen
+```
+
+For the HomeSeer devices specifically, you may need to update the `COMMAND_CLASS_CENTRAL_SCENE` for each node in your `zwcfg` file with the following:
```

Comment:
Let's move all the HomeSeer specific info into the [device-specific](https://github.com/home-assistant/home-assistant.github.io/blob/b0caec9c90f8d945f697c821fa83d54ab9dd2f67/source/_docs/z-wave/device-specific.markdown) page. (For reference: https://home-assistant.io/docs/z-wave/device-specific/ )

We can put a short note in here linking to that section.